pkg/operatorrules/compatibility_test.go

```diff
@@ -120,52 +120,36 @@ var _ = Describe("OperatorRules", func() {
 			Expect(registeredAlerts).To(ConsistOf(alerts))
 		})
 
-		It("should replace alerts with the same name in the same RegisterAlerts call", func() {
+		It("should replace alerts with the same name and same expression in the same RegisterAlerts call", func() {
```

Comment:
instead of "the same name" I would use "identical names"

Reply:
and what does it means replace? I think maybe this is more clear

"should register only unique expressions for alerts with identical names in a single RegisterAlerts call"

Reply:
identical is more "similar"
in this case, I think "same" is stronger and more accurate